### ISSUE TITLE
Grafana Cloud uses near-invisible grays in Dark Mode, or near-invisible colours in Light Mode

### ISSUE THREAD

**Issue by arielshaqed** [grafana/grafana#77189]
**Steps to reproduce**:

I am merely very short-sighted, other people probably have a much harder time.  Have someone with imperfect sight view the "explore" pane and a few metrics.

**Actual Result**:

The dark colour theme on Grafana Cloud looks like this:

![dark colour theme with near-invisible small gray test](https://github.com/grafana/grafana/assets/7802932/faf4222c-4240-43b5-9221-17460cdbe056)

Everything in gray is nearly invisible to me.  Everyone on my team uses dark mode, so I cannot sit with anyone and analyze logs.  The light colour theme looks like this:

![light colour theme makes it readable](https://github.com/grafana/grafana/assets/7802932/f829e3e6-be04-49f8-8d12-0bf054c09064)

But then colourized logs browsers created by the rest of the team become mostly invisible: they use escape sequences to set colours on messages, which does not switch well from dark to light mode.

**Expected Result**

1. Dark mode should provide contrast to be relevant for visually impaired users.
1. Logs browser should have a way to specify colours that will be sensitive to light/dark mode switch.

**Relevant WCAG Criteria:** [1.4.3 Contrast (minimum) - Level AA](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0#contrast-minimum)

THANKS!

**Environment**:
- Grafana version: Grafana v10.3.0-62488 (f6a0f6912f) on Cloud
- Data source type & version: Prometheus (metrics), Loki (logs)
- User OS & Browser: Fedora Linux, Firefox
- Others:

**Comment by ashharrison90** [comment]
@arielshaqed when you say "everything in grey", which bits are you referring to? 

as far as i can tell, everything there meets wcag AA criteria with the exception of the `Enter a PromQL query` placeholder text: 
![image](https://github.com/grafana/grafana/assets/20999846/c22d7d7f-b6ba-4aa4-88f1-cb3436f627de)

**Comment by eledobleefe** [comment]
@arielshaqed we are closing this issue as we recently merged an issue that solves the accessibility issue of the placeholder, the only one we could find that didn't meet the wcag AA criteria.
Feel free to leave a comment if you think it's not been fixed yet and we can reopen.

**Comment by arielshaqed** [comment]
> @arielshaqed we are closing this issue as we recently merged an issue that solves the accessibility issue of the placeholder, the only one we could find that didn't meet the wcag AA criteria. Feel free to leave a comment if you think it's not been fixed yet and we can reopen.

Thanks, looking forward to seeing it on the cloud!
